components/bookshelf/Navigation.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react"
+import { NavBox, NavList, Link } from "../styled/bookshelf"
+
+const Navigation = ({ active, setActive }) => {
+  const handleClick = (e: string | any) => {
+    setActive(e.target.innerText)
+  }
+  return (
+    <NavBox>
+      <NavList>
+        <Link
+          $active={active === "Reading List" ? "active" : ""}
```

Comment:
I've only seen the usage of the `$` sign in jQuery. I'm not sure this is good.

Reply:
ah, thats for the styled component prop. I just got it from StackOvereflow. It just adds the purple border if the prop is set to 'active'. Is there a better way to do this?

![image](https://user-images.githubusercontent.com/91857581/188564202-801f0235-1b62-484e-aae0-b31adade1b25.png)

Reply:
Hmm can you try removing the $ sign in the and just make it  `active`? It should work without it.